src/components/Popover/src/Popover.vue

```diff
@@ -0,0 +1,214 @@
+<template>
+	<div style="display: contents;">
+		<!-- @slot Element that the popover is tied or 'tethered' to. Must be a single node. -->
+		<slot
+			name="tether"
```

Comment:
Rename to "action" (consistent with Orbit).

I understand the popover is tethered to the content of this slot, but this slot does not contain the "tether". It contains the "button", "trigger", or "action" that the popover is tethered to.

Reply:
I was going back and forth on this too, my first reaction was also that `action` made more sense, but it might have been a bias since we used that term often in Orbit.

Reply:
Changed!